test/e2e/nodelatencymonitor_test.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	"antrea.io/antrea/pkg/features"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestNodeLatencyMonitor(t *testing.T) {
+	skipIfFeatureDisabled(t, features.NodeLatencyMonitor, true, false)
+	skipIfHasWindowsNodes(t) // as the feature does not have official windows support
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	//  1: create the NodeLatencyMonitor CR with a 10s ping interval
+	_, err = data.crdClient.CrdV1alpha1().NodeLatencyMonitors().Create(context.TODO(), &crdv1alpha1.NodeLatencyMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+		},
+		Spec: crdv1alpha1.NodeLatencyMonitorSpec{
+			PingIntervalSeconds: int32(10),
+		},
+	}, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("Failed to create NodeLatencyMonitor CR: %v", err)
+	}
+	t.Logf("NodeLatencyMonitor CR created successfully.")
+
+	//  2: Poll until(5min) NodeLatencyStats are reported correctly
+	ctx := context.Background()
+	err = wait.PollImmediate(time.Second, 300*time.Second, func() (bool, error) {
```

Comment:
I am not sure what kind of checks you are doing in the condition function. You are computing max latency and average latency, but then you just log them? This is the test so it should validate the contents of the `NodeLatencyStats` objects to make sure they are correct.

I would suggest the following:
- a first call to `wait.PollImmediate`, with a 30s timeout (after all the ping interval is set to 10s). This should validate that the `NodeLatencyStats` objects are available. There should be one per Node, and for each one, there should be one entry in `PeerNodeLatencyStats` for each Node. You can also make sure that `TargetIPLatencyStats` has the correct size (1 if cluster is IPv4-only or IPv6-only, 2 if cluster is dual-stack, and that each latency number is strictly positive.
- a second call to `wait.PollImmediate`, again with a 30s timeout, which validates that the stats are refreshed after some time. You should perform the same validations as for the first call to `wait.PollImmediate`, but also check that all the `LastSendTime` and `LastRecvTime` fields have been updated.